vitrina/datasets/templates/vitrina/datasets/detail.html

```diff
@@ -1,7 +1,242 @@
 {% extends "base.html" %}
 {% load i18n %}
 
-{% block content %}
+
 <h1>{{ object.title }}</h1>
 <p>{{ object.description|linebreaks }}</p>
-{% endblock content %}
+
+{% block pageTitle %} | {% if locale == "lt" %}{{ dataset.title }}{% else %}{% if dataset.titleEn != "" %}{{ dataset.titleEn }}{% else %}{{ dataset.title }}{% endif %}{% endif %}{% endblock %}
+{% block pageDescription %}{% if locale == "lt" %}{{ dataset.description | escape }}{% else %}{{ dataset.descriptionEn | escape }}{% endif %}{% endblock %}
+{% block pageAuthor %}{{ organization.title }}{% endblock %}
+{% block pageOgTitle %} | {{ dataset.title }}{% endblock %}
+{% block pageOgDescription %}{% if locale == "lt" %}{{ dataset.description | escape }}{% else %}{{ dataset.descriptionEn | escape }}{% endif %}{% endblock %}
```

Comment:
Remove translatable model logic from templates, this will come with https://github.com/atviriduomenys/katalogas/issues/61.